orc8r/cloud/go/blobstore/sql_test.go

```diff
@@ -30,29 +30,31 @@ import (
 func TestSqlBlobStorage_ListKeys(t *testing.T) {
 	happyPath := &testCase{
 		setup: func(mock sqlmock.Sqlmock) {
-			mock.ExpectQuery("SELECT \"key\" FROM network_table").
+			mock.ExpectQuery("SELECT network_id, type, \"key\", version FROM network_table").
 				WithArgs("network", "type").
 				WillReturnRows(
-					sqlmock.NewRows([]string{"key"}).AddRow("key1").AddRow("key2"),
+					sqlmock.NewRows([]string{"network_id", "type", "key", "version"}).
+						AddRow("network", "t1", "key1", 42).
+						AddRow("network", "t2", "key2", 43),
```

Comment:
@hcgatewood Please advise if this falls in line with our conversation about just injecting intraspection on the defaults for code reviewers. This test modification was made because the call to sql.Search() generated a different query then sql.ListKeys()did

Reply:
We want to get rid of the per-implementation ListKeys tests -- so get rid of this one and the one for the ent impl